manifest/manifest.py

```diff
@@ -43,9 +43,6 @@ def _included_items(self, include_types=None):
                 for path in self.local_changes.iterdeleted():
                     if path in paths:
                         del paths[path]
-                if item_type == "reftest":
-                    for path, items in self.local_changes.iterdeletedreftests():
-                        paths[path] -= items
```

Comment:
Because reftests can be removed by changing references without removing any files. Therefore a different system is needed to store the reftest changes. This is that system.